src/main/resources/de/markiewb/netbeans/plugin/git/toolbar/layer.xml

```diff
@@ -69,5 +69,73 @@
                 <attr name="position" intvalue="900"/>
             </file>
         </folder>
+        
```

Comment:
Just to add some more precisions, the GitRemote actions becomes available in Netbeans as soon as  C/C++ is activated. You dont need to have an active (open) remote C/C++ project for the bar to show up although the icons will be gray.

At that point, we need to have a look at issue #7 and try to hide the bars completly when a different or no versionning system is used.